src/Button/index.tsx

```diff
@@ -0,0 +1,64 @@
+import type { FC, PropsWithChildren } from "react";
+
+import "./index.scss";
+import clsx from "clsx";
+
+export interface ButtonProps {
+  /**
+   * The button's size.
+   */
+  size?: "sm" | "lg";
+
+  /**
+   * The button's color scheme.
+   */
+  color?: "black" | "white" | "dark-blue" | "red" | "blue-a";
+
+  /**
+   * Whether the button is rounded.
+   */
+  rounded?: boolean;
+
+  /**
+   * Aria label for the button.
+   */
+  ariaLabel?: string;
+
+  /**
+   * Handler to fire when the button is clicked.
+   */
+  onClick(event: React.MouseEvent<HTMLButtonElement>): void;
```

Comment:
For some reason, this declaration is hidden from the autodocs. Luckily, this version works:

```suggestion
  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
```